Terminal.Gui/Views/Menu.cs

```diff
@@ -1025,6 +1025,8 @@ public MenuBar (MenuBarItem [] menus) : base ()
 			WantMousePositionReports = true;
 			IsMenuOpen = false;
 
+			Added += MenuBar_Added;
```

Comment:
I think since `MenuBar_Added` is just a single line *and* you never `-=` the delegate, it's probably better to just do 

```cs
Added += (sender) => _initialCanFocus = CanFocus;
```

???

Reply:
I like this better. But this is even better:

```cs
private void MenuBar_Added (View sender)
{
	_prevCanFocus = CanFocus;
	Added -= MenuBar_Added;
}
```

"initial" implies the value is only relevant during initialization. But as coded, it is really "the previous version of CanFocus". Right?

Also, we should used `sender` vs `obj` to be more clear.